### PR TITLE
tests: use released image for awsgeneric-16.04

### PR DESCRIPTION
**Pull request by orndorffgrant**
## Test Steps

Try to run an awsgeneric-16.04 test before and after this. It should work after this commit.

### CODE REVIEW

File: features/cloud.py [hunk 1]
```diff
@@ -189,7 +189,12 @@ def locate_image_name(self, series: str) -> str:
         elif "pro" in self.machine_type:
             image_type = ImageType.PRO
 
-        return self.api.daily_image(release=series, image_type=image_type)
+        if daily:
+            return self.api.daily_image(release=series, image_type=image_type)
+        else:
+            return self.api.released_image(
```
Comment (lucasmoura) [inline features/cloud.py:195]: Should we alert somehow that we are defaulting to a non-daily image ?

Comment (orndorffgrant) [reply features/cloud.py:195]: ah good point - I'll add a log in there

Comment (orndorffgrant) [reply features/cloud.py:197]: updated!

**Review by aciba90 — APPROVED** [review]
LGTM, thanks!

**Review by renanrodrigo — APPROVED** [review]

**Comment by renanrodrigo** [comment]
If you would please cherry-pick this one to the release branch

**Pull request merged**